cvat-ui/src/reducers/interfaces.ts

```diff
@@ -569,3 +569,8 @@ export interface CombinedState {
     shortcuts: ShortcutsState;
     review: ReviewState;
 }
+
+export enum DimensionType {
```

Comment:
@manasars , we have similar definition in cvat-core/src/enums.js. What is the reason for that?

Reply:
As mentioned in earlier review inputs, we have similar implementation of ReviewStatus in both the places, hence we implemented similarly.

Reply:
@nmanovic Need to rewrite cvat-core in typescript to avoid this.